1.preprocess_data/preprocess_training_data.py

```diff
@@ -0,0 +1,154 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Training Movie Preprocessor
+# ### Preprocesses Training Movies from Aspera IDR downloader
+# 
+# #### Import libraries
+
+# In[1]:
+
+
+import imagej
+import pandas as pd
+import pathlib
+from IPython.utils.io import capture_output
+import os
+
+import PyBaSiC.pybasic as pybasic
+import numpy as np
+
+import skimage
+
+
+# ### Determine what frames are labled from each training movie
+
+# In[2]:
+
+
+#Read plates listed in features dataset to figure out which frames from each movie have training data
+#Save these training locations into training_frames file
+
+def save_training_wells(traingset_path, save_path):
+    data_list = []
```

Comment:
```suggestion
def save_training_wells(traingset_path, save_path):
    """
    Read plates listed in features dataset to figure out which frames from each movie have training data
    Save these training locations into training_frames file
    """
    data_list = []
```

It's typical to include function descriptions in a [docstring](https://peps.python.org/pep-0257/).

Reply:
Added to https://github.com/WayScience/mitocheck_data/pull/2/commits/bc2ddcf3cc589d0e903776a37990664cd00b2a50.
Changes included in Jupyter Notebook.